cli/tests/lib_runtime_api.ts

```diff
@@ -1,12 +1,12 @@
 const [errors, program] = await Deno.compile(
-  "main.ts",
+  "/main.ts",
   {
-    "main.ts": `document.getElementById("foo");`,
+    "/main.ts": `document.getElementById("foo");`,
```

Comment:
I'm strongly in favor of this change - this will resolve https://github.com/denoland/deno/issues/5620